### ISSUE TITLE
time consuming is very large! Much slower than tensorflow version.  

### ISSUE THREAD

**Issue by yangxuehang** [Kathrine94/EAST#4]
Hi，I used the code u provided. I found it that it takes 60+s training a step. which is much slower than that in tensorflow version. 
I used the same gpu device, only 1 gpu in both methods. same 16 threads to provide image batch. I wonder why the time differs with each other so large！Could u please give me some tips or explanations?
Thank u!

**Comment by janzd** [comment]
It takes about 60 seconds to train one epoch. The original tensorflow code prints out information about training steps, one step being one batch. So if there is 1000 training images and you use batch size of 16 images, then one epoch is about 63 training steps.

**Comment by lizzyYL** [comment]
@kurapan @yangxuehang 
 Hello, I have a  stange problem.
 When I train the model on icdar2015. The process shows that the loss  is decreasing. But when I use the model to predict the result. All the results are wrong, somethings look like random big rectangles.

Did you have met this problem?Look forward to your reply :)
